.github/ISSUE_TEMPLATE/DOCS.yml

```diff
@@ -1,8 +1,10 @@
 name: "Documentation Update"
 description: Create an issue for documentation changes
 title: "Docs: <title>"
-labels: ["Documentation :books:"]
-projects: ["litestar-org/16"]
+labels:
```

Comment:
Do we need to change these lines? It basically is the same value, but different syntax.

Reply:
Whatever other maintainers decide :)
Thanks for working on it!